Buildings/Fluid/Movers/UsersGuide.mo

```diff
@@ -437,62 +446,276 @@ If the motor is cooled by the fluid, as indicated by
+Eu=(pressure forces)/(inertial forces)
 </p>
+from which one can derive the ratio of Euler numbers as<br/>
+<p align=\"center\" style=\"font-style:italic;\">
+Eu &frasl; Eu<sub>p</sub>=(&Delta;p&nbsp;V&#775;<sub>p</sub><sup>2</sup>)
+&frasl; (&Delta;p<sub>p</sub>&nbsp;V&#775;<sup>2</sup>).
+</p>
+<p>
+For more information, see
+<a href=\"Modelica://Buildings.Fluid.Movers.BaseClasses.Euler.correlation\">
+Buildings.Fluid.Movers.BaseClasses.Euler.correlation</a>
+and <a href=\"https://energyplus.net/assets/nrel_custom/pdfs/pdfs_v9.6.0/EngineeringReference.pdf\">
+EnergyPlus 9.6.0 Engineering Reference</a>
+chapter 16.4 equations 16.209 through 16.218.<br/>
+See
+<a href=\"Modelica://Buildings.Fluid.Movers.Validation.PowerEuler\">
+Buildings.Fluid.Movers.Validation.PowerEuler</a>
+as an example.
+</li>
+<li>
+<code>NotProvided</code> - The information of this efficiency term is not provided.
+During simulation, it will be computed using the other efficiency terms.
+</li>
+</ul>
 <p>
-However, if <code>per.use_powerCharacteristic=false</code>, then
-performance data for
-<i>&eta;<sub>hyd</sub></i> and
- <i>&eta;<sub>mot</sub></i> need to be provided by the user, and hence
-the model computes
+<i>&eta;<sub>mot</sub></i> can be computed using the following choices:
 </p>
-<p align=\"center\" style=\"font-style:italic;\">
-  &eta; = &eta;<sub>hyd</sub> &nbsp; &eta;<sub>mot</sub><br/>
-  P<sub>ele</sub> = W<sub>flo</sub> &frasl; &eta;.
+<ul>
+<li>
+<code>Efficiency_VolumeFlowRate</code>
+</li>
+<li>
+<code>Efficiency_MotorPartLoadRatio</code> - An array of efficiency vs. motor part load ratio
+<i>y<sub>mot</sub>=P<sub>ele</sub> &frasl; P<sub>ele,nominal</sub></i>
+is provided. During simulation, the efficiency is interpolated or extrapolated by
+<a href=\"Modelica://Buildings.Fluid.Movers.BaseClasses.Characteristics.efficiency_yMot\">
+Buildings.Fluid.Movers.BaseClasses.Characteristics.efficiency_yMot</a>.
+See
+<a href=\"Modelica://Buildings.Fluid.Movers.BaseClasses.Validation.FlowMachineInterface_yMot\">
+Buildings.Fluid.Movers.BaseClasses.Validation.FlowMachineInterface_yMot</a>
+as an example.
+</li>
+<li>
+<code>GenericCurve</code> - The rated motor power input and maximum motor
+efficiency are provided. The model then uses a generic motor efficiency curve
+as a function of motor part load ratio generated by
+<a href=\"Modelica://Buildings.Fluid.Movers.BaseClasses.Characteristics.motorEfficiencyCurve\">
+Buildings.Fluid.Movers.BaseClasses.Characteristics.motorEfficiencyCurve</a>.
+If a power curve is provided, the maximum power value on this curve with a 20%
+oversize would be used as default.
+The model then computes the efficiency the same way as the option of
+<code>Efficiency_MotorPartLoadRatio</code>.
+Also see
+<a href=\"Modelica://Buildings.Fluid.Movers.BaseClasses.Validation.FlowMachineInterface_yMot\">
+Buildings.Fluid.Movers.BaseClasses.Validation.FlowMachineInterface_yMot</a>
+as an example.
+</li>
+<li>
+<code>NotProvided</code>
```

Comment:
This item misses the description, please add it

Reply:
Added in df96e8233e3b1cc7bb2fd7285a1813900805191f.